src/runtime/composables-v5/useStrapi.ts

```diff
@@ -0,0 +1,101 @@
+import type { FetchOptions } from 'ofetch'
+
+import type { Strapi5ResponseSingle, Strapi5RequestParams, Strapi5ResponseMany } from '../types/v5'
+import { useStrapiVersion, useStrapiClient } from '#imports'
+
+interface StrapiV5Client<T> {
+  find<F = T>(contentType: string, params?: Strapi5RequestParams): Promise<Strapi5ResponseMany<F>>
+  findOne<F = T>(contentType: string, docuemntId?: string | Strapi5RequestParams, params?: Strapi5RequestParams): Promise<Strapi5ResponseSingle<F>>
```

Comment:
Why not keep the old signature here? The `id?: string | number | Strapi5RequestParams` seems fine to me.